wgpu-types/src/lib.rs

```diff
@@ -536,19 +536,26 @@ bitflags::bitflags! {
 }
 
 #[repr(C)]
-#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
 pub struct BufferDescriptor {
+    pub label: *const std::os::raw::c_char,
     pub size: BufferAddress,
     pub usage: BufferUsage,
 }
 
 #[repr(C)]
-#[derive(Clone, Debug, Default, PartialEq, Eq, Hash)]
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
 pub struct CommandEncoderDescriptor {
     // MSVC doesn't allow zero-sized structs
     // We can remove this when we actually have a field
-    pub todo: u32,
+    // pub todo: u32,
+    pub label: *const std::os::raw::c_char,
+}
+
+impl Default for CommandEncoderDescriptor {
+    fn default() -> CommandEncoderDescriptor {
+        unsafe { std::mem::zeroed() }
```

Comment:
why use zeroed instead of just assigning `ptr::null()` safely?

Reply:
I can't remember at this point, but `mem::zeroed` is safe in this context. All zeros is a valid bit pattern for `CommandEncoderDescriptor` as it exists in this PR. However, I could see how someone could accidentally make it UB later (e.g. if for some reason a reference field was added).

I'll change it to use `ptr::null`.